torch/csrc/jit/script/init.cpp

```diff
@@ -524,9 +538,10 @@ void initJitScriptBindings(PyObject* module) {
     .def("graph_for", [](Method& self, py::args args, py::kwargs kwargs) {
       return self.graph_for(createStackForSchema(self.getSchema(), std::move(args), std::move(kwargs)));
     })
-    .def("forward_schema", [](Method &self, Def &def, bool is_method) {
+    .def("forward_schema", [](Method &self, Def &def, py::object defaults, bool is_method) {
```

Comment:
nit: `(Method &self, Def &def, const std::unordered_map<std::string, py::object>& defaults, bool is_method)`
enables you to remove the cast, and will improve the error message if this is called incorrectly.